src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -564,6 +565,23 @@
 					Click="DiskCleanupButton_Click"
 					Content="{helpers:ResourceString Name=DiskCleanup}" />
 
+				<!--  Format Disk  -->
+				<TextBlock
```

Comment:
Can you wrap this in a settings control?

Reply:
![image](https://github.com/files-community/Files/assets/102259289/edacfe6e-bc12-4b50-93cb-ec83cbb1ba7c)

What icons should we use?
One might be this one:
![image](https://github.com/files-community/Files/assets/102259289/3d0fcd31-4291-4b2c-aa3e-4e0c118d65ab)

Reply:
It's probably best to not use any icon. As a side note, if you use the button style, the whole card will act as a button.

Reply:
![image](https://github.com/files-community/Files/assets/102259289/ca2144bb-e46d-44ad-b2d4-d29859513717)

Reply:
Nice! Can you move them out of the container so that they are by themselves?

Reply:
Can you also add an icon like this?
![image](https://github.com/files-community/Files/assets/39923744/f6411094-d4af-4e26-99e3-94acfb7df4a7)

Reply:
![image](https://github.com/files-community/Files/assets/102259289/0eb54f1e-b621-45c4-ae8e-760b55652f50)